docs/development.md

```
@@ -132,6 +132,9 @@ My Dockerized instance:
 My VS Code instance:
 ![VS Code instance](res/dev/devinst-13.png)
 
+## RFC Notes (Host IDE + Docker Execution)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)